frame/collective/src/lib.rs

```diff
@@ -292,36 +195,22 @@ decl_module! {
 		///
 		/// NOTE: Does not enforce the expected `MaxMembers` limit on the amount of members, but
 		///       the weight estimations rely on it to estimate dispatchable weight.
-		///
-		/// # <weight>
-		/// ## Weight
-		/// - `O(MP + N)` where:
-		///   - `M` old-members-count (code- and governance-bounded)
-		///   - `N` new-members-count (code- and governance-bounded)
-		///   - `P` proposals-count (code-bounded)
-		/// - DB:
-		///   - 1 storage mutation (codec `O(M)` read, `O(N)` write) for reading and writing the members
-		///   - 1 storage read (codec `O(P)`) for reading the proposals
-		///   - `P` storage mutations (codec `O(M)`) for updating the votes for each proposal
-		///   - 1 storage write (codec `O(1)`) for deleting the old `prime` and setting the new one
-		/// # </weight>
-		#[weight = (
+		#[pallet::weight(
 			T::WeightInfo::set_members(
 				*old_count, // M
 				new_members.len() as u32, // N
 				T::MaxProposals::get() // P
-			),
-			DispatchClass::Operational
```

Comment:
same here, we just need to put the same tuple inside the syntax: pallet::weight()

Reply:
Correction Done :+1: